Cargo.toml

```diff
@@ -23,6 +23,10 @@ futures = { version = "0.3", optional = true }
 tun-tap = { version = "0.1.3", optional = true }
 etherparse = { version = "0.13.0", optional = true }
 
+[dependencies.gat-std]
+optional = true
+version = "0.1.1"
```

Comment:
For consistency, please make this dependency one line, like the other dependencies above and put it under the line with the `futures` package.